### DISCUSSION TITLE
Showing Bin colour as a seperate attribute?

### DISCUSSION THREAD

**Discussion by GrantH-H** [mampfes/hacs_waste_collection_schedule#3024] [Q&A]
I've probably done something dumb in my setup. I've had the sensor configured and working for a while but ive started looking at it as I'm trying to streamline my dashboard a bit. 

In my instance if I look at the waste collection sensor it returns this: 
`{{states('sensor.waste_collection')}} - Blue Bin in 2 days`

Under developer tools if i look at sensor.waste_collection i see 

```
state: 
Blue Bin in 2 days

State attributes (YAML, optional)
DATE_TEMPLATE: Grey Bin
daysTo: 2
attribution: "Last update: 11/13/24 01:00:31"
icon: mdi:trash-can
friendly_name: waste_collection
```

So I'm not sure where the grey bin is coming from or how I can add a separate attribute that would show just Blue Bin?

**Comment by 5ila5** [comment]
Adding additional attributes to sensors is not really supported (apart from adding daysTo and changing [details format](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md#options-for-details_format-parameter))

But you can create additional sensors and modify the value_template to fit your needs (`'{{value.types|join(", ")}}'` to only show collection type, comma separated if there are multiple at the same day)

You can add and modify your sensors by going to setting -> devices & services -> Waste Collection Schedule -> select customize on the source you want to add/modify sensors -> select the sensor to modify or add one -> change the attributes as needed
